App.Loans/Repositories/LoansRepository.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using App.Loans.Models;
+using App.Configuration;
+using App.Repositories;
+
+namespace App.Loans.Repositories
+{
+    public interface ILoansRepository
+    {
+        IEnumerable<Loan> GetAll();
+        IEnumerable<string> GetValues();
+
+        Loan Get(int id);
+    }
+    public class LoansRepository : ILoansRepository, ITransientDependency
+    {
+        private Loan[] loans = { new Loan(10000, 10, 24), new Loan(28860, 3, 6), new Loan(5000, 0.01, 3) };
```

Comment:
Please, place each single statement on new line. It will make code much easier to read